src/content-helper/editor-sidebar/title-suggestions/title-suggestions.scss

```diff
@@ -100,56 +104,88 @@
 		display: flex;
 		flex-direction: column;
 		align-items: flex-start;
-		gap: 4px;
+		gap: 8px;
```

Comment:
Some missing `to_rem()`s in this file? Maybe `gap` doesn't work with `to_rem()`? Seeing this for some other values below as well.